src/main/java/com/aspectsecurity/contrast/contrastjenkins/Constants.java

```diff
@@ -11,4 +11,7 @@
     public static final String VULNERABILITY_STATUS_BEING_TRACKED = "Being+Tracked";
     public static final String VULNERABILITY_STATUS_UNTRACKED = "Untracked";
     public static final String VULNERABILITY_STATUS_NOT_A_PROBLEM = "NotAProblem";
+
+    public static final int defaultAppVersionTagFormat = 1;
```

Comment:
constants should be CAPS_WITH_UNDERSCORES

Reply:
refactored